subsys/logging/log_backend_rtt.c

```diff
@@ -10,39 +10,49 @@
 #include <logging/log_output.h>
 #include <SEGGER_RTT.h>
 
-#define DROP_MAX 99
+#ifndef CONFIG_LOG_BACKEND_RTT_BUFFER_SIZE
+#define CONFIG_LOG_BACKEND_RTT_BUFFER_SIZE 0
+#endif
+
+#ifndef CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE
+#define CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE 0
+#endif
+
+#ifndef CONFIG_LOG_BACKEND_RTT_OUTPUT_BUFFER_SIZE
+#define CONFIG_LOG_BACKEND_RTT_OUTPUT_BUFFER_SIZE 0
+#endif
 
-#if CONFIG_LOG_BACKEND_RTT_MODE_DROP
+#ifndef CONFIG_LOG_BACKEND_RTT_RETRY_DELAY_MS
+/* Long enough to detect host presence */
+#define CONFIG_LOG_BACKEND_RTT_RETRY_DELAY_MS 10
+#endif
+
+#ifndef CONFIG_LOG_BACKEND_RTT_RETRY_CNT
+/* Big enough to detect host presence */
+#define CONFIG_LOG_BACKEND_RTT_RETRY_CNT 10
+#endif
+
+#define DROP_MAX 99
 
 #define DROP_MSG "\nmessages dropped:    \r"
```

Comment:
I know that it is not part of this change, but do you know what is a purpose of `\r` at the end?

Reply:
nope,